module-services/service-evtmgr/EventManager.cpp

```diff
@@ -321,6 +321,10 @@ int EventManagerCommon::dumpLogsToFile()
 
 void EventManagerCommon::handleMinuteUpdate(time_t timestamp)
 {
+    // temporary frequency lock to handle time update event
+    if (cpuSentinel) {
+        cpuSentinel->HoldMinimumFrequencyForTime(bsp::CpuFrequencyMHz::Level_4, std::chrono::seconds(1));
```

Comment:
I think that we can go lower than Level 4 here - as I checked, the redraw + update for eink took max 1.5s when on 24MHz clock

Reply:
done